Tasks/ANT/anttask.ts

```diff
@@ -78,14 +79,17 @@ function processAntOutputLine(line) {
 
 async function doWork() {
 
-    function enableCodeCoverage() {
+    function enableCodeCoverage(): Q.Promise<any> {
```

Comment:
This task is becoming unreadable.  Why can't enableCodeCoverage be in another module?